### ISSUE TITLE
Build common packages into sys.so 

### ISSUE THREAD

**Issue by ViralBShah** [JuliaLang/JuliaBox#123]
We can dramatically cut down the time of loading Gadfly and many other packages by simply including something like the following in `userimg.jl`. Dependencies get compiled automatically. I tried this out and it works:
````
include("/Users/viral/.julia/v0.3/Gadfly/src/Gadfly.jl")
importall Gadfly
````

To start with, we could just include Gadfly and PyPlot.

@staticfloat Is it possible to rebuild the `sys.so` from your PPAs, or should we just build from source?

**Comment by tanmaykm** [comment]
Closed via #172

**Comment by ViralBShah** [comment]
This is worth announcing on the list!
